__tests__/app.test.js

```diff
@@ -94,7 +94,7 @@ describe("3: GET /api/reviews/review_id", () => {
   });
 });
 
-/*describe("4: GET /api/:review_id/comments", () => {
+describe("4: GET /api/:review_id/comments", () => {
   test("status:200, responds with an array of comments", () => {
     const review_id = 1;
```

Comment:
This is a good test but having another test for when you provide a valid review_id that has no comments associated to it. What status code should this return? what would you expect to receive as a response from this?